lib/fnsapi/tmp_storage.rb

```diff
@@ -13,6 +13,7 @@ def write_token(token, expire_at)
     end
 
     def token
+      @file = File.open(file_path, 'a+')
```

Comment:
We create the `@file` variable in object initializer. Why do you want to rewrite it here?

Reply:
https://github.com/actie/fnsapi/blob/master/lib/fnsapi/base_service.rb#L48 here we use the `token` method for the `TmpStorage` class instance, which initialized here: https://github.com/actie/fnsapi/blob/master/lib/fnsapi/base_service.rb#L41

So the `@file` variable should already exist for this instance.